lib/ecto/association.ex

```diff
@@ -883,14 +884,9 @@ defmodule Ecto.Association.BelongsTo do
 
   @doc false
   def struct(module, name, opts) do
-    ref       = if ref = opts[:references], do: ref, else: :id
-    queryable = Keyword.fetch!(opts, :queryable)
-    related   = Ecto.Association.related_from_query(queryable, name)
-
-    unless is_atom(related) do
-      raise ArgumentError, "association queryable must be a schema, got: #{inspect related}"
-    end
-
+    ref        = if ref = opts[:references], do: ref, else: :id
+    queryable  = Keyword.fetch!(opts, :queryable)
+    related    = Ecto.Association.related_from_query(queryable, name)
```

Comment:
If the lines are going to be changed anyway, I'd prefer to remove the alignment instead of adding more of it